<launch>

  <node pkg="ccny_rgbd" type="sparse_tracker_f_node" name="sparse_tracker_f" 
    output="screen">

    <param name="base_frame"   value="camera_link"/>
    <param name="fixed_frame"  value="odom"/>

    <param name="history_size"          value="10"/>

    <param name="n_orb_features"        value="400"/>

    <param name="max_range"             value="10.0"/>
    
    <param name="icp_max_iterations"    value="25"/>
    <param name="icp_max_corresp_dist"  value="0.15"/>
    <param name="icp_tf_epsilon"        value="1e-15"/>

    <param name="vgf_features_res"      value="0.07"/>

    <param name="use_value_rejection"     value="false"/>
    <param name="max_value_diff"          value = "50.0"/>
    
    <param name="use_ransac_rejection"    value="true"/>
    <param name="ransac_inlier_threshold" value="0.20"/>

    <param name="min_features" value="10"/>

    <param name="orb_smooth" value="0"/>
    <param name="orb_window" value="0"/>

    <param name="ransac_matching_distance" value="1000.0"/>
    <param name="ransac_inlier_threshold" value="0.30"/>

  </node>

  <node pkg="ccny_3dmap_server" type="map_server_node" name="map_server" 
    output="screen">

    <param name="fixed_frame"  value="odom"/>
    <param name="resolution"   value="0.02"/>
    <param name="max_range"  value="5"/>

  </node>

</launch>

